packages/webapp/src/containers/Finances/selectors.js

```diff
@@ -23,7 +24,33 @@ const selectedSaleSelector = createSelector(financeSelector, (state) => state.se
 
 const expenseSelector = createSelector(financeSelector, (state) => state.expenses);
 
-const expenseTypeSelector = createSelector(financeSelector, (state) => state.expense_types);
+const allExpenseTypeSelector = createSelector(financeSelector, (state) => state.expense_types);
+
+const expenseTypeSelector = createSelector(financeSelector, (state) => {
+  return state.expense_types.filter((type) => !type.deleted);
+});
+
+const expenseTypeTileContentsSelector = createSelector(financeSelector, (state) => {
+  const defaultTypes = [];
+  const customTypes = [];
+  state.expense_types.forEach((type) => {
+    if (!type.deleted) {
+      const arrayToUpdate = type.farm_id ? customTypes : defaultTypes;
+      arrayToUpdate.push(type);
+    }
+  });
+
+  return [
+    ...defaultTypes.sort((typeA, typeB) =>
+      i18n
+        .t(`expense:${typeA.expense_translation_key}`)
+        .localeCompare(i18n.t(`expense:${typeB.expense_translation_key}`)),
+    ),
+    ...customTypes.sort((typeA, typeB) =>
+      typeA.expense_translation_key.localeCompare(typeB.expense_translation_key),
```

Comment:
The logic was moved from `ExpenseCategories` component.
I followed the same pattern as `taskType` (sorted default types -> sorted custom types):
https://github.com/LiteFarmOrg/LiteFarm/blob/0dced6cb926689a6b625b0307f88f3def0e10426/packages/webapp/src/components/Task/PureTaskTypeSelection/PureTaskTypeSelection.jsx#L121-L176